### DISCUSSION TITLE
Spirit Bear(Lone Druid) separate courier action to deliver items directly to Spirit Bear

### DISCUSSION THREAD

**Discussion by MatviyPodolsky** [ValveSoftware/Dota2-Gameplay#10891] [Ideas]
To make Lone Druid gameplay more comfortable I'm suggesting to add separate action for courier to deliver items to summon directly. It's annoying to manage items delivery one by one via drag-n-drop to Spirit Bear, while he is moving or fighting. 
Also courier lose delivery target if Spirit Bear ability level was upgraded(courier just fly to position where Spirit Bear was upgraded)
As additional issue is managing courier in Turbo mode. You need to use shift+"return to base" command after drag-n-drop items to you summon. In other case courier will not return to base after delivery, and you will not be able to send it manually(only by delivering next item)

**Comment by victormarcel09** [comment]
Implementing a dedicated action to send items directly to the Spirit Bear would save players time and effort, especially during moments of combat or movement. This streamlined process could make managing items on the Spirit Bear more intuitive and efficient.
[EZ Drive MA](https://www.paybyplatema.me/)
